app/hotels/src/allHotels/searchForm/locationPicker/__generated__/LocationPickerScreen_cities_Query.graphql.js

```diff
@@ -0,0 +1,217 @@
+/**
```

Comment:
@tbergq Now we have two `LocationPickerScreen_cities_Query.graphql.js` files. It obviously doesn't work very well if you are only moving the files...

Reply:
wtf, I specifically searched for this filename, vscode found only one 😖

Reply:
@mrtnzlml  should be deleted now 🙏